pkg/controller/deployment/cert_manager_cainjector_deployment.go

```diff
@@ -29,6 +29,7 @@ var (
 		"cert-manager-deployment/cainjector/cert-manager-cainjector-leaderelection-rb.yaml",
 		"cert-manager-deployment/cainjector/cert-manager-cainjector-leaderelection-role.yaml",
 		"cert-manager-deployment/cainjector/cert-manager-cainjector-sa.yaml",
+		"cert-manager-deployment/cainjector/cert-manager-cainjector-svc.yaml",
```

Comment:
Introduced by https://github.com/cert-manager/cert-manager/commit/9273158452faa8dcb2df009883dac68d329467a9 (release v1.16.0)

Reply:
Add this helps resolve the original bug: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_cert-manager-operator/275/pull-ci-openshift-cert-manager-operator-master-e2e-operator/1924672586587836416/artifacts/e2e-operator/gather-extra/artifacts/services.json has `cert-manager-cainjector` service in latest e2e run.